starlite/_signature/parsing.py

```diff
@@ -145,6 +152,11 @@ def parse_fn_signature(
     parsed_params: list[ParsedSignatureParameter] = []
     dependency_names: set[str] = set()
     fn_type_hints = get_fn_type_hints(fn)
+    return_annotation = fn_type_hints.get("return", signature.empty)
+
+    owner_is_generic_controller = owner and isinstance(owner, GenericController)
```

Comment:
```suggestion
    owner_is_generic_controller = isinstance(owner, GenericController)
```

should be good enough